--- lineak_defaultplugin-0.8.4/default_plugin/Makefile.am.orig	2005-07-28 17:43:46.000000000 +0200
+++ lineak_defaultplugin-0.8.4/default_plugin/Makefile.am	2006-03-07 21:04:03.210423000 +0100
@@ -20,9 +20,9 @@
 DEFS = @DEFS@ -DCONFDIR=\"@sysconfdir@\" 
 
 install-data-local:
-	$(mkinstalldirs) $(sysconfdir)/
-	$(INSTALL_DATA) $(top_srcdir)/media-detect.conf $(sysconfdir)/media-detect.conf
+	$(mkinstalldirs) $(DESTDIR)$(sysconfdir)/
+	$(INSTALL_DATA) $(top_srcdir)/media-detect.conf $(DESTDIR)$(sysconfdir)/media-detect.conf
 
 uninstall-local:
-	-rm -f $(sysconfdir)/media-detect.conf
+	-rm -f $(DESTDIR)$(sysconfdir)/media-detect.conf
 
